virtme/commands/run.py

```diff
@@ -947,7 +947,7 @@ def console_client(args):
     with open(console_script_path, "w", encoding="utf-8") as file:
         print(
             (
-                "#! /bin/bash\n"
+                "#!/usr/bin/env bash\n"
```

Comment:
I'm not sure if it is really needed in the VM with only Linux, nor about the consequences. Maybe best to drop this patch for the moment?

Reply:
Sure, we can postpone the whole patch for now.